features/storage/TESTS/filesystem/general_filesystem/main.cpp

```diff
@@ -1896,7 +1898,9 @@ static void FS_append_non_empty_file()
 
     res = !((fd[0] = fopen("/default/" "filename", "a+")) != NULL);
     TEST_ASSERT_EQUAL(0, res);
-
+#if defined(__MICROLIB)
```

Comment:
Could you add a comment to say that microlib does not support opening a file in an append mode?